Podverse/PVDeleter.swift

```diff
@@ -94,12 +99,11 @@ class PVDeleter: NSObject {
                         PVDownloader.shared.decrementBadge()
                         
                         if shouldCallNotificationMethod == true {
-                            NotificationCenter.default.post(name: .episodeDeleted, object: nil, userInfo: ["mediaUrl":mediaUrl as? Any])
+                            NotificationCenter.default.post(name: .episodeDeleted, object: nil, userInfo: ["mediaUrl":mediaUrl ?? ""])
```

Comment:
I had this idea that Notifications are supposed to always get dispatched to the main thread I think. Do we want this notification on the global thread?